website/project/model.py

```diff
@@ -2992,8 +3011,15 @@ class Sanction(StoredObject):
     REJECTION_NOT_AUTHORIZED_MESSAEGE = 'This user is not authorized to reject this {DISPLAY_NAME}'
     REJECTION_INVALID_TOKEN_MESSAGE = 'Invalid rejection token provided for this {DISPLAY_NAME}.'
 
-    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+    # Controls whether or not the Sanction needs unanimous approval or just a single approval
+    ANY = 'any'
+    UNANIMOUS = 'unanimous'
+    mode = UNANIMOUS
+
     initiation_date = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow)
```

Comment:
For future reference, this can be simplified to `auto_now_add=True` in modularODM (in which case utcnow is the default: see  https://github.com/CenterForOpenScience/modular-odm/blob/develop/modularodm/fields/datetimefield.py#L45-45)